src/main/java/TaskManager.java

```diff
@@ -0,0 +1,146 @@
+import java.util.ArrayList;
+
+public class TaskManager {
+
+    public ArrayList<Task> Tasks;
+    public TaskManager() {
+        this.Tasks = new ArrayList<Task>();
+    }
+
+    private static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    private static final String LINEBREAK = "____________________________________________________________";
+    public void storeTasks(String text) {
+        System.out.println(LINEBREAK);
+        String[] result = text.split(" ");
+        String[] result2 = text.split("/by ");
+        String[] result3 = text.split("/at ");
```

Comment:
bad SLAP here; remember the usage of specific methods in the `TaskS` repo used in Lecture 6? that should give you an idea how to improve SLAP.